core/src/test/scala/com/mesosphere/usi/core/integration/SchedulerIntegrationTest.scala

```diff
@@ -0,0 +1,51 @@
+package com.mesosphere.usi.core.integration
+
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Keep
+import com.mesosphere.mesos.conf.MesosClientSettings
+import com.mesosphere.usi.core.Scheduler
+import com.mesosphere.usi.core.helpers.SchedulerStreamTestHelpers.{outputFlatteningSink, specInputSource}
+import com.mesosphere.usi.core.launching.SimpleShellCommandInfoGenerator
+import com.mesosphere.usi.core.matching.ScalarResourceRequirement
+import com.mesosphere.usi.core.models._
+import com.mesosphere.utils.AkkaUnitTest
+import com.mesosphere.utils.mesos.MesosClusterTest
+import org.apache.mesos.v1.Protos
+import org.scalatest.Inside
+
+class SchedulerIntegrationTest extends AkkaUnitTest with MesosClusterTest with Inside {
+  "launches the thing" in {
+    implicit val materializer = ActorMaterializer()
+
+    val settings = MesosClientSettings(mesosFacade.url)
+    val frameworkInfo = Protos.FrameworkInfo.newBuilder().setUser("test").setName("SimpleHelloWorldExample").build()
+
+    val client = Scheduler.connect(settings, frameworkInfo).futureValue
+
+    val podId = PodId("pod-1")
```

Comment:
Can we please not use the `pod1` and `app1` names? Please keep them unique and readable (take Marathon ITs as an example)

Reply:
✅